manifests/service-binding-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[{"apiVersion":"apps.openshift.io/v1alpha1","kind":"ServiceBindingRequest","metadata":{"name":"example-servicebindingrequest"},"spec":{"applicationSelector":{"matchLabels":{"connects-to":"postgres","environment":"production"},"resourceKind":"Deployment"},"backingSelector":{"resourceName":"database.example.org","resourceVersion":"v1alpha1"}}}]'
+    capabilities: Basic Install
+    containerImage: quay.io/redhat-developer/service-binding-operator:v0.0.1
+    description: An operator to support binding capabilities between imported apps and operator backed services
+  name: service-binding-operator.v0.0.1
+  namespace: placeholder
+spec:
+  links:
+  - name: Source Code
+    url: https://github.com/redhat-developer/service-binding-operator
+  maintainers:
+    - email: shbose@redhat.com
+      name: Shoubhik Bose
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: ServiceBindingRequest
+      name: servicebindingrequests.apps.openshift.io
+      version: v1alpha1
+  description: Placeholder description
+  displayName: Operator Service Binding Operator
+  install:
+    spec:
+      deployments:
+      - name: service-binding-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: service-binding-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: service-binding-operator
+            spec:
+              containers:
+              - command:
+                - service-binding-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: service-binding-operator
+                image: quay.io/redhat-developer/service-binding-operator:v0.0.1
```

Comment:
@sbose78 Is this version number correct?

Reply:
v0.0.4

Reply:
Done